src/actions/firestore.js

```diff
@@ -302,7 +302,10 @@ export function setListeners(firebase, dispatch, listeners) {
   // Only attach one listener (count of matching listener path calls is tracked)
   if (config.oneListenerPerPath) {
     return listeners.forEach(listener => {
-      const path = getQueryName(listener);
+      const path =
+        getQueryName(listener) +
```

Comment:
We may want to modify `getQueryName` to return `storeAs` as the query name if it exists.

I believe this is already happening in the v1.0.0-alpha version, but have to check to be sure.

Reply:
Open to getting a change in for this, just want to make sure we get it in the right place.

Reply:
This makes sense, because the query name is the same for different listeners.  Plus, it's consistent with the identifiers elsewhere (like data, ordered, and composite).

Reply:
And the current getQueryName doesn't account for pagination options like `startAt` or `startAfter`, so without `storeAs` the previous page of results stored in Redux gets overwritten rather than the new data appended.  `storeAs` is required to work around this.

Reply:
It appears unsetting the listener using `storeAs` will need some refactoring.

Reply:
I submitted a PR for this #182